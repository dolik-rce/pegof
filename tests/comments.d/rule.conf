input comments.d/rule.peg
