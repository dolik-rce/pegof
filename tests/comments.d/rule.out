# This is grammar comment

# This is rule comment for 'main'
main <-
    A
    / B
    / C
    / D

# This is weirdly indented comment for rule 'A'
A <- "A"

# This is multiline
# comment for rule 'B'
B <- "B"

# This is not really well placed comment for rule 'C'
C <- "C"

# These are
#
# comments for rule "D"
#
# that are separated by empty lines
D <- "D"
